source/how-tos/app-development/interactive/form.rst

```diff
@@ -222,6 +222,24 @@ auto_modules_<MODULE>
      attributes:
        auto_modules_matlab:
          default: false
+
+  To filter versions that show up in the drop-down list, you can use a Ruby regex (https://rubular.com/)
+  (without the wrapping `//`) or a string:
+
+  Regex:
+  .. code-block:: yaml
```

Comment:
I think you need a newline between these 2, this code-block isn't rendering right.

![image](https://github.com/OSC/ood-documentation/assets/4874123/6f1bfe6f-a2cb-4a8d-8ddc-5c3bad0c33ed)